app/main.py

```diff
@@ -131,11 +136,10 @@ def grab_secrets(self):
                     cert_file.write(self._cert_to_pem(cert.cer))
 
     def _dump_pfx(self, pfx, cert_filename, key_filename):
-        import base64
         from OpenSSL import crypto
         p12 = crypto.load_pkcs12(base64.decodestring(pfx))
         pk = crypto.dump_privatekey(crypto.FILETYPE_PEM, p12.get_privatekey())
-        cert = crypto.dump_certificate(crypto.FILETYPE_PEM, p12.get_certificate())
+        certs = (p12.get_certificate(),) + p12.get_ca_certificates() or ()
```

Comment:
you must put `p12.get_ca_certificates() or ()` in brackets, otherwise it might fail if `get_ca_certificates()` return `None`

so it should look like this:
```python
certs = (p12.get_certificate(),) + (p12.get_ca_certificates() or ())
```